tezos_interop/tezos_interop.re

```diff
@@ -11,6 +11,16 @@ let rec try_decode_list = (l, string) =>
   | [] => None
   };
 
+let blake2b_20_encoding =
+  Data_encoding.(
+    conv(
+      hash => BLAKE2B_20.to_raw_string(hash) |> Bytes.of_string,
+      // TODO: I don't like this exception below
```

Comment:
What exception?

Reply:
Option.get in the line below

Reply:
Can you elaborate your comment (in the file)?